### DISCUSSION TITLE
Note regarding 1password-cli command op installation path

### DISCUSSION THREAD

**Discussion by andrewcrook** [Homebrew/discussions#4348] [Everyday usage]
### Output of `brew config`

```shell
n/a
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

Since recent upgrade (formula version 2.13.1) my shell couldn’t find the op command 
It seems that the location is now 

`/usr/local/bin/`

to fix you need to add it to your PATH

`/opt/homebrew/Caskroom/1password-cli/2.15.0/`

just contains the installation package

**Comment by ZPrimed** [comment]
To follow-on to this - I just migrated from an Intel to Apple Silicon Mac, and the whole "Homebrew lives in /opt on ARM" thing threw me a bit. Once I realized what was happening, I started going through and cleaning up all of /usr/local on the new ARM Mac (since I know that was all from Migration Assistant pulling it off my old Intel machine). 

But the `op` binary apparently still wants to live in `/usr/local/bin` on Apple Silicon (as Andrew said above), so you can't just wholescale nuke the contents of `/usr/local`.

**Comment by gromgit** [comment]
> It seems that the location is now
> 
> `/usr/local/bin/`
> 
> to fix you need to add it to your PATH

`/usr/local/bin` is a well-known and oft-used directory for locally-installed binaries across the Unix world. That's why even macOS retains it as part of its base `PATH` setting:
```
monterey% env -i /usr/libexec/path_helper
PATH="/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin"; export PATH;
```
Removing it from your `PATH` would be a Bad Idea indeed. That's why `brew shellenv` _prepends_ `/opt/homebrew/bin` to your PATH, instead of replacing `/usr/local/bin` with it.
